python/lesson1/tutorial.md

```diff
@@ -66,65 +89,66 @@ Everybody's favourite pastime.
 
 ### Combining Operations
 
 Being able to only perform one operation at a time is pretty limiting, so Python allows us to combine mathematical operations. Try this one:
 
-    9 * 4 - 6
+    >>> 9 * 4 - 6
 
 Now try a few more. You can combine as many operations as you like.
 
 ### Operator Precedence
 
 Did you notice any unexpected results when you started combining operations? If you didn't, try this:
 
-    10 - 2 * 4
+    >>> 10 - 2 * 4
 
-Python follows the traditional mathematical rules of precedence, which state that multiplication and division are done before addition and subtraction. (You may remember *BODMAS*.) This means in our example above, 2 and 4 are multiplied first, and then the result is subtracted from 10.
+Python follows the traditional mathematical rules of precedence, which state that multiplication and division are done before addition and subtraction. (You may remember *[BODMAS](https://en.wikipedia.org/wiki/BODMAS)*.) This means in our example above, 2 and 4 are multiplied first, and then the result is subtracted from 10.
 
 We can change the order of operations by using parentheses. Anything inside parentheses is executed first.
 
 Now try it like this:
 
-    (10 - 2) * 4
+    >>> (10 - 2) * 4
 
 You should have a different answer.
 
 Because of precedence rules, complex operations such as our first example can be quite confusing to read. If you find yourself writing more complex expressions, there is no harm in adding parentheses for clarity.
 
 ### Decimal Points
 
-Remember dividing 14 by 4? The result probably surprised you. Fortunately, there is a way to convince Python that you really do want the full answer, not just the whole number. We can do this by *starting* with numbers with decimal points:
+One of the things that tends to confuse newcomers to programming in general is the concept of *floating point numbers*.  Basically, numbers with decimal points tend to behave a little strangely when you're performing mathematical operations on them.  The reasons for this are complex and rooted in the nature of computing itself, so for now, let's just go with the assumption that weird things happen with decimal numbers and leave it at that.
 
-    14.0 / 4.0
+To see an example of this, try dividing `10` by `3`:
 
-Now try a few more and see how it goes. Remember that you can also use non-whole numbers in your calculations.
+    >>> 10 / 3
 
-You may notice that there are still some limitations. For example, try:
+The answer should go on forever, but it doesn't.  Now try something a little more precision-sensitive:
 
-    10.0 / 3.0
+    >>> 1.000000000000001 * 8
 
-The answer should go on forever, but it doesn't. This is because there is only a finite amount of space to store the number, and it runs out after approximately 15 significant figures.
+Probably not what you'd expect right?  For now, you'll just have to accept this as a limitation, and later on you'll learn how other programmers work around it.
```

Comment:
To be honest, *I don't actually know why this happens*.  Just that it has something to do with how numbers are stored in memory, limitations of physics or something.  I've always just been happy to live with the mystery, but if you have an explanation we can put in here I'm all ears.